pkg/artifacts/signable.go

```diff
@@ -33,7 +33,8 @@ type Signable interface {
 	StorageBackend(cfg config.Config) sets.String
 	Signer(cfg config.Config) string
 	PayloadFormat(cfg config.Config) formats.PayloadType
-	Key(interface{}) string
+	FullKey(interface{}) string
```

Comment:
bikeshed: I'm okay with Key/ShortKey. I know we've been lax about this, but we should also add godoc describing the difference between the 2 (you can generalize instead of going into the specifics of each impl).

Reply:
We want docs here as well since this is where Signable implementors / IDEs are going to look for documentation for the type.

Reply:
Oh I see. Added! Please take a look! Thanks